<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>接口测试整合版</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 120%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">接口测试整合版</h1>
<p class='attribute'><strong>测试人员 : </strong> 小A</p>
<p class='attribute'><strong>开始时间 : </strong> 2019-11-30 19:21:34</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:00.450026</p>
<p class='attribute'><strong>测试结果 : </strong> 共 65，错误 65，通过率= 0.00%</p>

<p class='description'>读取配置来操作代码</p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 0.00% }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 0 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 0 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 65 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 16px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='errorClass warning'>
    <td>testqcd_01_register.TestRegister</td>
    <td class="text-center">10</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">10</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',10)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='ft1_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_01: 成功注册-完整</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse in" align="left">
    <pre>
    
ft1_1: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_02: 成功注册-无昵称</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse in" align="left">
    <pre>
    
ft1_2: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_03: 成功注册-昵称超过10</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse in" align="left">
    <pre>
    
ft1_3: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_04: 密码为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse in" align="left">
    <pre>
    
ft1_4: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_05: 手机号为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_5'>错误</button>
    <div id='div_ft1_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_5'>错误</button>
    <div id='div_ft1_5' class="collapse in" align="left">
    <pre>
    
ft1_5: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_06: 手机号-不足11位</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse in" align="left">
    <pre>
    
ft1_6: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_07: 手机号-超过11位</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse in" align="left">
    <pre>
    
ft1_7: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_8' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_08: 密码-少于8位</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse in" align="left">
    <pre>
    
ft1_8: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_9' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_09: 密码-多于16位</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse in" align="left">
    <pre>
    
ft1_9: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_10' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_10: 手机号已注册</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse in" align="left">
    <pre>
    
ft1_10: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_01_register.py", line 38, in test_register
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>testqcd_02_login.TestLogin</td>
    <td class="text-center">5</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">5</td>
    <td class="text-center"><a href="javascript:showClassDetail('c2',5)" class="detail" id='c2'>详细</a></td>
</tr>

<tr id='ft2_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_1: 正常登录</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse in" align="left">
    <pre>
    
ft2_1: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_02_login.py", line 37, in test_login
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_2: 手机号为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_2'>错误</button>
    <div id='div_ft2_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_2'>错误</button>
    <div id='div_ft2_2' class="collapse in" align="left">
    <pre>
    
ft2_2: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_02_login.py", line 37, in test_login
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_3: 密码为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_3'>错误</button>
    <div id='div_ft2_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_3'>错误</button>
    <div id='div_ft2_3' class="collapse in" align="left">
    <pre>
    
ft2_3: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_02_login.py", line 37, in test_login
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_4: 密码错误</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_4'>错误</button>
    <div id='div_ft2_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_4'>错误</button>
    <div id='div_ft2_4' class="collapse in" align="left">
    <pre>
    
ft2_4: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_02_login.py", line 37, in test_login
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_5: 手机号错误</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_5'>错误</button>
    <div id='div_ft2_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_5'>错误</button>
    <div id='div_ft2_5' class="collapse in" align="left">
    <pre>
    
ft2_5: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_02_login.py", line 37, in test_login
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>testqcd_03_rechargd.TestRecharge</td>
    <td class="text-center">16</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">16</td>
    <td class="text-center"><a href="javascript:showClassDetail('c3',16)" class="detail" id='c3'>详细</a></td>
</tr>

<tr id='ft3_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_01: 不登录-充值</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_1'>错误</button>
    <div id='div_ft3_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_1'>错误</button>
    <div id='div_ft3_1' class="collapse in" align="left">
    <pre>
    
ft3_1: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_02: 正常登录</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_2'>错误</button>
    <div id='div_ft3_2' class="collapse in" align="left">
    <pre>
    
ft3_2: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_03: 充值成功-整数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_3'>错误</button>
    <div id='div_ft3_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_3'>错误</button>
    <div id='div_ft3_3' class="collapse in" align="left">
    <pre>
    
ft3_3: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_04: 充值成功-1位小数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_4'>错误</button>
    <div id='div_ft3_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_4'>错误</button>
    <div id='div_ft3_4' class="collapse in" align="left">
    <pre>
    
ft3_4: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_05: 充值成功-2位小数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_5'>错误</button>
    <div id='div_ft3_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_5'>错误</button>
    <div id='div_ft3_5' class="collapse in" align="left">
    <pre>
    
ft3_5: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_06: 充值成功-50万</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_6'>错误</button>
    <div id='div_ft3_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_6'>错误</button>
    <div id='div_ft3_6' class="collapse in" align="left">
    <pre>
    
ft3_6: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_07: 会员id为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_7'>错误</button>
    <div id='div_ft3_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_7'>错误</button>
    <div id='div_ft3_7' class="collapse in" align="left">
    <pre>
    
ft3_7: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_8' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_08: 会员id不存在</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_8'>错误</button>
    <div id='div_ft3_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_8'>错误</button>
    <div id='div_ft3_8' class="collapse in" align="left">
    <pre>
    
ft3_8: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_9' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_09: 会员id为字符串</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_9'>错误</button>
    <div id='div_ft3_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_9'>错误</button>
    <div id='div_ft3_9' class="collapse in" align="left">
    <pre>
    
ft3_9: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_10' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_10: 会员id为小数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_10'>错误</button>
    <div id='div_ft3_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_10'>错误</button>
    <div id='div_ft3_10' class="collapse in" align="left">
    <pre>
    
ft3_10: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_11' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_11: 充值金额为0</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_11'>错误</button>
    <div id='div_ft3_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_11'>错误</button>
    <div id='div_ft3_11' class="collapse in" align="left">
    <pre>
    
ft3_11: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_12' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_12: 充值金额为负数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_12'>错误</button>
    <div id='div_ft3_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_12'>错误</button>
    <div id='div_ft3_12' class="collapse in" align="left">
    <pre>
    
ft3_12: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_13' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_13: 充值金额为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_13'>错误</button>
    <div id='div_ft3_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_13'>错误</button>
    <div id='div_ft3_13' class="collapse in" align="left">
    <pre>
    
ft3_13: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_14' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_14: 充值金额-3位小数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_14'>错误</button>
    <div id='div_ft3_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_14'>错误</button>
    <div id='div_ft3_14' class="collapse in" align="left">
    <pre>
    
ft3_14: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_15' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_15: 充值金额-大于50万</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_15'>错误</button>
    <div id='div_ft3_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_15'>错误</button>
    <div id='div_ft3_15' class="collapse in" align="left">
    <pre>
    
ft3_15: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_16' class='none'>
    <td class='errorCase'><div class='testcase'>test_rechargd_16: 充值金额-非数字</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_16'>错误</button>
    <div id='div_ft3_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_16'>错误</button>
    <div id='div_ft3_16' class="collapse in" align="left">
    <pre>
    
ft3_16: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_03_rechargd.py", line 34, in test_rechargd
    row=cases.caseid+1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>testqcd_04_add.TestAdd</td>
    <td class="text-center">16</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">16</td>
    <td class="text-center"><a href="javascript:showClassDetail('c4',16)" class="detail" id='c4'>详细</a></td>
</tr>

<tr id='ft4_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_01: 不登录-加标</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_1'>错误</button>
    <div id='div_ft4_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_1'>错误</button>
    <div id='div_ft4_1' class="collapse in" align="left">
    <pre>
    
ft4_1: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_02: 正常登录</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_2'>错误</button>
    <div id='div_ft4_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_2'>错误</button>
    <div id='div_ft4_2' class="collapse in" align="left">
    <pre>
    
ft4_2: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_03: 加标成功</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_3'>错误</button>
    <div id='div_ft4_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_3'>错误</button>
    <div id='div_ft4_3' class="collapse in" align="left">
    <pre>
    
ft4_3: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_04: 借款人id为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_4'>错误</button>
    <div id='div_ft4_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_4'>错误</button>
    <div id='div_ft4_4' class="collapse in" align="left">
    <pre>
    
ft4_4: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_05: 标题为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_5'>错误</button>
    <div id='div_ft4_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_5'>错误</button>
    <div id='div_ft4_5' class="collapse in" align="left">
    <pre>
    
ft4_5: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_06: 借款金额为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_6'>错误</button>
    <div id='div_ft4_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_6'>错误</button>
    <div id='div_ft4_6' class="collapse in" align="left">
    <pre>
    
ft4_6: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_07: 年利率为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_7'>错误</button>
    <div id='div_ft4_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_7'>错误</button>
    <div id='div_ft4_7' class="collapse in" align="left">
    <pre>
    
ft4_7: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_8' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_08: 借款期限为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_8'>错误</button>
    <div id='div_ft4_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_8'>错误</button>
    <div id='div_ft4_8' class="collapse in" align="left">
    <pre>
    
ft4_8: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_9' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_09: 借款期限为字符串</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_9'>错误</button>
    <div id='div_ft4_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_9'>错误</button>
    <div id='div_ft4_9' class="collapse in" align="left">
    <pre>
    
ft4_9: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_10' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_10: 借款期限类型为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_10'>错误</button>
    <div id='div_ft4_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_10'>错误</button>
    <div id='div_ft4_10' class="collapse in" align="left">
    <pre>
    
ft4_10: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_11' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_11: 竞标天数为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_11'>错误</button>
    <div id='div_ft4_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_11'>错误</button>
    <div id='div_ft4_11' class="collapse in" align="left">
    <pre>
    
ft4_11: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_12' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_12: 借款人id为负数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_12'>错误</button>
    <div id='div_ft4_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_12'>错误</button>
    <div id='div_ft4_12' class="collapse in" align="left">
    <pre>
    
ft4_12: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_13' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_13: 借款人id为小数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_13'>错误</button>
    <div id='div_ft4_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_13'>错误</button>
    <div id='div_ft4_13' class="collapse in" align="left">
    <pre>
    
ft4_13: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_14' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_14: 借款人id不存在</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_14'>错误</button>
    <div id='div_ft4_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_14'>错误</button>
    <div id='div_ft4_14' class="collapse in" align="left">
    <pre>
    
ft4_14: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_15' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_15: 借款金额为负数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_15'>错误</button>
    <div id='div_ft4_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_15'>错误</button>
    <div id='div_ft4_15' class="collapse in" align="left">
    <pre>
    
ft4_15: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft4_16' class='none'>
    <td class='errorCase'><div class='testcase'>test_add_16: 借款金额为小数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_16'>错误</button>
    <div id='div_ft4_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_16'>错误</button>
    <div id='div_ft4_16' class="collapse in" align="left">
    <pre>
    
ft4_16: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_04_add.py", line 33, in test_add
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>testqcd_05_invest.TestInvest</td>
    <td class="text-center">18</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">18</td>
    <td class="text-center"><a href="javascript:showClassDetail('c5',18)" class="detail" id='c5'>详细</a></td>
</tr>

<tr id='ft5_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_01: 借款人正常登录</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_1'>错误</button>
    <div id='div_ft5_1' class="collapse in" align="left">
    <pre>
    
ft5_1: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_02: 借款人加标</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_2'>错误</button>
    <div id='div_ft5_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_2'>错误</button>
    <div id='div_ft5_2' class="collapse in" align="left">
    <pre>
    
ft5_2: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_03: 管理员正常登录</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_3'>错误</button>
    <div id='div_ft5_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_3'>错误</button>
    <div id='div_ft5_3' class="collapse in" align="left">
    <pre>
    
ft5_3: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_04: 管理员审核</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_4'>错误</button>
    <div id='div_ft5_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_4'>错误</button>
    <div id='div_ft5_4' class="collapse in" align="left">
    <pre>
    
ft5_4: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_05: 投资人正常登录</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_5'>错误</button>
    <div id='div_ft5_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_5'>错误</button>
    <div id='div_ft5_5' class="collapse in" align="left">
    <pre>
    
ft5_5: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_06: 投资人正常投资</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_6'>错误</button>
    <div id='div_ft5_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_6'>错误</button>
    <div id='div_ft5_6' class="collapse in" align="left">
    <pre>
    
ft5_6: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_07: 投资人id为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_7'>错误</button>
    <div id='div_ft5_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_7'>错误</button>
    <div id='div_ft5_7' class="collapse in" align="left">
    <pre>
    
ft5_7: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_8' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_08: 标id为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_8'>错误</button>
    <div id='div_ft5_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_8'>错误</button>
    <div id='div_ft5_8' class="collapse in" align="left">
    <pre>
    
ft5_8: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_9' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_09: 投资金额为空</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_9'>错误</button>
    <div id='div_ft5_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_9'>错误</button>
    <div id='div_ft5_9' class="collapse in" align="left">
    <pre>
    
ft5_9: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_10' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_10: 投资人id为负数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_10'>错误</button>
    <div id='div_ft5_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_10'>错误</button>
    <div id='div_ft5_10' class="collapse in" align="left">
    <pre>
    
ft5_10: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_11' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_11: 投资人id为字符串</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_11'>错误</button>
    <div id='div_ft5_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_11'>错误</button>
    <div id='div_ft5_11' class="collapse in" align="left">
    <pre>
    
ft5_11: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_12' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_12: 标id为负数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_12'>错误</button>
    <div id='div_ft5_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_12'>错误</button>
    <div id='div_ft5_12' class="collapse in" align="left">
    <pre>
    
ft5_12: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_13' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_13: 标id为字符串</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_13'>错误</button>
    <div id='div_ft5_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_13'>错误</button>
    <div id='div_ft5_13' class="collapse in" align="left">
    <pre>
    
ft5_13: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_14' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_14: 投资金额为负数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_14'>错误</button>
    <div id='div_ft5_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_14'>错误</button>
    <div id='div_ft5_14' class="collapse in" align="left">
    <pre>
    
ft5_14: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_15' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_15: 投资金额为不能被100整除的数</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_15'>错误</button>
    <div id='div_ft5_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_15'>错误</button>
    <div id='div_ft5_15' class="collapse in" align="left">
    <pre>
    
ft5_15: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_16' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_16: 投资人id不存在</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_16'>错误</button>
    <div id='div_ft5_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_16'>错误</button>
    <div id='div_ft5_16' class="collapse in" align="left">
    <pre>
    
ft5_16: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_17' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_17: 标id不存在</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_17' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_17'>错误</button>
    <div id='div_ft5_17' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_17' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_17'>错误</button>
    <div id='div_ft5_17' class="collapse in" align="left">
    <pre>
    
ft5_17: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_18' class='none'>
    <td class='errorCase'><div class='testcase'>test_invest_18: 投资金额大于项目的剩余可投金额</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_18' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_18'>错误</button>
    <div id='div_ft5_18' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_18' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_18'>错误</button>
    <div id='div_ft5_18' class="collapse in" align="left">
    <pre>
    
ft5_18: Traceback (most recent call last):
  File "E:\py37\APITest\libs\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\py37\APITest\cases\testqcd_05_invest.py", line 38, in test_invest
    row = cases.caseid + 1
AttributeError: 'CaseDate' object has no attribute 'caseid'


    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>65</td>
    <td>0</td>
    <td>0</td>
    <td>65</td>
    <td>通过率：0.00%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
